packages/shared/src/lib/analytics.ts

```diff
@@ -169,6 +170,7 @@ export enum FeedItemTitle {
 }
 
 export enum TargetType {
+  PromotionCard = 'promotion_card',
```

Comment:
We have this field already `MarketingCtaCard`

Reply:
Yeah, I noticed it was available but the name for the enum was quite specific to the case of marketing, so I thought of separating them. Though I can unify them to avoid duplicates by renaming the original one.

Reply:
Updated here: https://github.com/dailydotdev/apps/pull/3174/commits/98359bebfc225e4ea7eb7963df3ed521c7ff8f97 🥳